modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -465,12 +465,6 @@ def test_astype():
     expected_df_casted = expected_df.astype(dtype_dict)
     df_equals(modin_df_casted, expected_df_casted)
 
-    # Ignore lint because this is testing bad input
-    bad_dtype_dict = {"index": np.int32, "index": np.int64, "index": str}  # noqa F601
```

Comment:
This is just like: `modin_df.astype({"index": str})`. I don't see the point in testing it.